sample-app/src/components/cards/StandardCard.tsx

```diff
@@ -41,48 +41,47 @@ interface CtaData {
   linkType: string
 }
 
+function isCtaData(data: unknown): data is CtaData {
```

Comment:
Should this function return a `boolean`? Or does the `is CtaData` syntax tell the IDE/TS the wrapped data is `CtaData` and therefore we get field autocompletion?

Reply:
Correct, it's a type guard which tells TS whether or not the `data` is a `CtaData`. The underlying javascript has this function simply returning a boolean. You can think of `data is CtaData` as a boolean which TS is able to get some extra info from.